control/pzmap.py

```diff
@@ -75,16 +106,20 @@ def pzmap(sys, Plot=True, title='Pole Zero Map'):
 
     if (Plot):
         import matplotlib.pyplot as plt
+
+        if isdtime(sys):
+            zgrid(plt)
```

Comment:
I think we should make plotting the zgrid an option, rather than the default.  This will give consistency with past behavior.

Reply:
I am actually having it similar to what we have on rlocus function, with an argument grid=False (even though IMHO plot is way better with the omega-damping grid)

I also realized we should also be using this on rlocus function, so I am making zgrid available for rlocus and making _sgrid_func() from rlocus available for pzmap.

Reply:
Yes, this sounds like a good approach.  And if we can reuse an existing function, that is even better (although I note that `_sgrid_func()` is not all that well commented -:).